language/en-GB/en-GB.plg_fabrik_form_workflow.ini

```
@@ -94,7 +94,6 @@ PLG_FORM_WORKFLOW_DELETE_RECORD_EMAIL="Report Abuse"
 
 PLG_FORM_WORKFLOW_DELETE_RECORD_LIST="Delete"
 PLG_FORM_WORKFLOW_REPORT_RECORD_LIST="Report"
```

pq essa linha desapareceu? Adicione novamente

```suggestion
PLG_FORM_WORKFLOW_REPORT_EDIT_RECORD_LIST="Suggest edit"
```

---

PLG_FORM_WORKFLOW_REPORT_EDIT_RECORD_LIST="Suggest edit"

---

Corrigido